cs464-group-project/src/pages/Players/PlayersMainPage.js

```diff
@@ -62,52 +66,102 @@ export function Player() {
     fetchPlayers();
   }, []);
 
+  async function selectTeam(event) {
+    setSelectedTeam(event.target.value);
+    const team = event.target.value;
+
+    try {
+      if (team === 'Random') {
+        setPlayers(randomPlayers);
+      } else {
+        if (teamcache[team]) {
+          setPlayers(teamcache[team]);
+        } else {
+          const response = await getAllPlayersByTeam(team); // Fetch players for the selected team
+
+          if (!response.ok) {
+            setPlayers(response.player || []);
+            teamcache = response.player;
```

Comment:
getting an warning saying 'teamchace' is constant.  If we're changing the variable value, would a 'let teamcache' work instead?

Reply:
Should be fixed now